deploy/contents/install/app/install.sh

```diff
@@ -683,11 +683,14 @@ if is_service_requested cp-docker-registry; then
         fi
 
         print_info "-> Push base tools images into the docker registry"
-        if [ -z $CP_POINT_IN_TIME_CONFIGURATION_DIR ]; then 
-           CP_DOCKER_MANIFEST_PATH=${CP_DOCKER_MANIFEST_PATH:-"$INSTALL_SCRIPT_PATH/../../dockers-manifest"}
+        
+        is_data_is_available_in_point_in_time_configuration tools
```

Comment:
why you re-build path to the file here if your `is_data_is_available_in_point_in_time_configuration` function returns it?